doc/source/migrations/migrate0_1.rst

```diff
@@ -47,17 +47,21 @@ and will be removed in MontePy 1.0.0.
 New Interface & Migration
```

Comment:
Can we include a code block showing a before/after? (MontePy 0.x / MontePy 1.0)

Reply:
RIP. That's a good idea, but a lot of the processes don't map really. But... yeah good idea.